.gitignore

```diff
@@ -25,3 +25,4 @@ _test_plugins_fail/*.so
 opentracingplugin/build
 build/
 skptesting/lorem.html
+.vscode/*
```

Comment:
i am generally rather against including editor specific references, even in .gitignore. OTOH, I can see that there are already some items that are not related to the provided source code, like .idea. @szuecs @AlexanderYastrebov WDYT?

Reply:
I've configured to ignore `.idea/` globally using https://stackoverflow.com/a/7335487/828328
I think this makes more sense than adding it to any new cloned project's `.gitignore`